packages/clerk-js/src/ui/components/OrganizationProfile/InviteMembersForm.tsx

```diff
@@ -182,8 +183,26 @@ export const InviteMembersForm = (props: InviteMembersFormProps) => {
   );
 };
 
+/**
+ * Determines default role from the organization settings or fallback to
+ * the only available role.
+ */
+const useDefaultRole = () => {
+  const { options } = useFetchRoles();
+  const { organizationSettings } = useEnvironment();
+
+  let defaultRole = organizationSettings.domains.defaultRole;
+
+  if (!defaultRole && options?.length === 1) {
```

Comment:
Shouldn't this be >1 ?

"If default role doesn't exist and fetched roles exist, use the first fetched role"

Reply:
We're checking if there's only one role returned from FAPI here, instead of selecting the first one. This avoids making an assumption on behalf of the user, as there could be cases where they wouldn't want that one the first one as default.